### PR TITLE
Added Gallery app

### LINKED ISSUE #83: Create app for Gallery
This app would be used to display items under nav-bar item `Photo Gallery`.

Display all gallery images at `/gallery/` and the slideshow after selecting any image.

### PR DESCRIPTION
**Pull request by geekyJock8**
Fixes #83 #14 #68 
Please set the size of the "thumbnail" in photo sizes in admin panel to 200x200 for better viewing. 
Also since I'm unable to deploy please do a local deployment. Sorry for the inconvenience 😞.

### CODE REVIEW

**Comment by Monal5031** [comment]
@geekyJock8 sir Please fix the codacy issues.

File: wsgi/odyssy/basic/static/basic/css/style.css [hunk 1]
```diff
@@ -472,139 +472,88 @@ nav ul li{
...
+ 	margin: 1rem 0;
+ 	display: block;
+ }
+
+
+ .button{
+ 	color: white;
+ 	margin: 1rem 0;
+ 	background-color: rgb(15,108,107);
+    border: none;
```
Comment (yashLadha) [inline wsgi/odyssy/basic/static/basic/css/style.css]: Fix the indentation

**Comment by Monal5031** [comment]
@yashLadha @Prakash2403 @mohitkyadav @aashutoshrathi Please review `more` depends on this one.

File: wsgi/odyssy/gallery/models.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from __future__ import unicode_literals
+
+
```
Comment (aashutoshrathi) [inline wsgi/odyssy/gallery/models.py]: This file can be removed.

File: wsgi/odyssy/gallery/tests.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (aashutoshrathi) [inline wsgi/odyssy/gallery/tests.py]: This can be removed too.

**Pull request merged**